docs/components/dropdown/README.md

```diff
@@ -188,6 +188,26 @@ Create a split dropdown button, where the left button provides standard
 <!-- dropdown-split.vue -->
 ```
 
+## Hidden Toggle
```

Comment:
Technically this isn't really a hide toggle, as toggle refers to the button that triggers the dopdown.

This probably should be called `hide-caret`, as removing the class `dropdown-toggle` stops the caret from being rendered (which is generated by CSS pseudo elements to add the dropdown/dropup caret)

Reply:
Actually as @mosinve suggested, `no-caret` would be better as the prop name.